test/slow/nvm run/Running "nvm run --silent" should work

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```

Comment:
this file doesn't seem to be running in https://travis-ci.org/creationix/nvm/jobs/166615285 (expand the "Adding APT Sources" section) - can you `chmod a+x` it?